webapp/pages/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Head from 'next/head';
+import { LoginForm } from 'dogma/features/auth/LoginForm';
+
+const HomePage = () => (
+  <>
+    <Head>
+        <link rel="icon" href="favicon.ico" />
+        <meta
+          name="Central Dogma"
+          content="Let's store your config in Central Dogma"
+        />
+    </Head>
+    <LoginForm />
```

Comment:
- Should we hide `<Layout>` component for `<LoginForm>`?
- If a session has been created, should we make `<LoginForm>` not be exposed to `HomePage`?

Reply:
Thanks, updated.